src/apps_plugin.c

```diff
@@ -2427,31 +2467,41 @@ int main(int argc, char **argv)
 
 	parse_args(argc, argv);
 
-	all_pids = calloc(sizeof(struct pid_stat *), (size_t) pid_max);
+	all_pids = calloc((size_t)pid_max, sizeof(struct pid_stat *));
 	if(!all_pids) {
-		error("Cannot allocate %lu bytes of memory.", sizeof(struct pid_stat *) * pid_max);
+		error("Cannot allocate %zu bytes of memory.", sizeof(struct pid_stat *) * pid_max);
 		printf("DISABLE\n");
 		exit(1);
 	}
 
-	fprintf(stdout, "CHART netdata.apps_cpu '' 'Apps Plugin CPU' 'milliseconds/s' apps.plugin netdata.apps_cpu stacked 140000 %d\n", update_every);
-	fprintf(stdout, "DIMENSION user '' incremental 1 %d\n", 1000);
-	fprintf(stdout, "DIMENSION system '' incremental 1 %d\n", 1000);
-
-	fprintf(stdout, "CHART netdata.apps_files '' 'Apps Plugin Files' 'files/s' apps.plugin netdata.apps_files line 140001 %d\n", update_every);
-	fprintf(stdout, "DIMENSION files '' incremental 1 1\n");
-	fprintf(stdout, "DIMENSION pids '' absolute 1 1\n");
-	fprintf(stdout, "DIMENSION fds '' absolute 1 1\n");
-	fprintf(stdout, "DIMENSION targets '' absolute 1 1\n");
+	#pragma GCC diagnostic push
+	#pragma GCC diagnostic ignored "-Wformat"
+	fprintf(stdout, "CHART netdata.apps_cpu '' 'Apps Plugin CPU' 'milliseconds/s' apps.plugin netdata.apps_cpu stacked 140000 %d\n"
+					"DIMENSION user '' incremental 1 1000\n"
+					"DIMENSION system '' incremental 1 1000\n"
+					"CHART netdata.apps_files '' 'Apps Plugin Files' 'files/s' apps.plugin netdata.apps_files line 140001 %1$d\n"
+					"DIMENSION files '' incremental 1 1\n"
+					"DIMENSION pids '' absolute 1 1\n"
+					"DIMENSION fds '' absolute 1 1\n"
+					"DIMENSION targets '' absolute 1 1\n",
+			update_every);
+	#pragma GCC diagnostic pop
```

Comment:
ok, something new! positional arguments in `printf()`.
Does the compiler complain without these `#pragma`?